src/qrl/core/Wallet.py

```diff
@@ -145,7 +145,7 @@ def decrypt_item(self, index: int, key: str):
         if index < len(self._address_items):
             cipher = AESHelper(key)
             tmp = self._address_items[index]._asdict()  # noqa
-            tmp['address'] = cipher.decrypt(tmp['address']).decode()
+            tmp['qaddress'] = cipher.decrypt(tmp['qaddress']).decode()
```

Comment:
this will break compatibility with webwallet

Reply:
added some code in Wallet.save_wallet() and Wallet._get_address_item_from_json() to help with that